lte/gateway/c/oai/tasks/sgw_s8/sgw_s8_handlers.c

```diff
@@ -505,6 +507,31 @@ void sgw_s8_handle_create_session_response(
         session_rsp_p->cause, session_rsp_p->context_teid);
     sgw_remove_sgw_bearer_context_information(
         sgw_state, session_rsp_p->context_teid, imsi64);
+  } else {
+    char teidString[16]             = {0};
+    char imsi_str[IMSI_BCD_DIGITS_MAX + 1] = {0};
+    IMSI64_TO_STRING(imsi64, (char*) imsi_str, IMSI_BCD_DIGITS_MAX);
+    spgw_ue_context_t* ue_context_p = NULL;
+    hashtable_ts_get(
+        sgw_state->imsi_ue_context_htbl, (const hash_key_t) imsi64,
+        (void**) &ue_context_p);
+    if (ue_context_p) {
+      sgw_s11_teid_t* s11_teid_p = NULL;
+      LIST_FOREACH(s11_teid_p, &ue_context_p->sgw_s11_teid_list, entries) {
```

Comment:
can you add a comment saying this generates a comma separated list of sgw Control Plane Teids 
Can you mention if those TEIDs will be only for default bearers?

Reply:
Always control plane teids are created for default bearer, So I would like to mention in the comment that update control plane teid to directoryd for each pdn session.